packages/itwinui-css/backstop/tests/progress-indicator.html

```diff
@@ -25,9 +25,10 @@
     </script>
     <style>
       @import url("./assets/demo.css") layer(demo);
+      @import url('@itwin/itwinui-variables') layer(variables);
       @import url("@itwin/itwinui-css/css/all.css") layer(itwinui);
       section[id^='demo'] {
-        width: 33vw;
+        width: 35vw;
```

Comment:
Why changed?

Reply:
this was done to prevent line wrapping as the sizes are now slightly bigger